controllers/settings.strorefronts.js

```diff
@@ -10,6 +11,46 @@ const get = async (req, res) => {
   return res.status(HTTPStatus.OK).send(settings.values)
 }
 
+const put = async (req, res) => {
+
+  const transaction = await sequelize.transaction()
+
+  try {
+    const settings = await models.SettingsStoreFronts.findOne({
+      where: { name: req.params.name },
+      transaction,
+    })
+
+    if (!settings) {
+      await models.SettingsStoreFronts.create(
+        {
+          name: req.params.name,
+          values: req.body,
+        },
+        {
+          where: { name: req.params.name },
```

Comment:
`create` does not accept a `where` clause (and `returning` is implicit as well, if I'm not mistaken)